tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -142,9 +148,26 @@ def test_node_maintenance(
         assert typed_nodes, f"Failed to find a {node_type} node for the test"
         typed_node_name = typed_nodes[0].name
 
+        # check csi-cephfsplugin-provisioner's and csi-rbdplugin-provisioner's
+        # are ready, see BZ #2162504
+        provis_pods = get_pods_having_label(
+            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
+            defaults.ROOK_CLUSTER_NAMESPACE,
+        )
+        provis_pods += get_pods_having_label(
+            constants.CSI_RBDPLUGIN_PROVISIONER_LABEL,
+            defaults.ROOK_CLUSTER_NAMESPACE,
+        )
+        provis_pod_names = [p["metadata"]["name"] for p in provis_pods]
+
         # Maintenance the node (unschedule and drain)
         drain_nodes([typed_node_name])
 
+        # avoid scenario when provisioners yet not been created (6 sec for creation)
+        retry(ResourceNotFoundError, tries=2, delay=2, backoff=2)(
```

Comment:
Why wait for so little? I think it will be better to increase the timeout by a few seconds(maybe change the delay to `5` or `10`).

Reply:
I was consulting with @ypersky1980 about performance. From her resp in comments to bz https://bugzilla.redhat.com/show_bug.cgi?id=2162504 , creation of PVC is 0.2-1 sec. Thus creation of provisioner pods will be even lesser. I don't want to give more time than necessary for this operation.

Reply:
also the wait_for_pods_to_be_running timeout is 200 sec but not 6 sec from retry. 6 sec is for pods creation - I added them more for safety than from necessity

Reply:
Okay, that sounds good.